agimus_demos_common/launch/franka_hardware.launch.py

```diff
@@ -28,7 +28,11 @@ def launch_setup(
         executable="ros2_control_node",
         parameters=[
             franka_controllers_params,
-            {"arm_id": arm_id, "load_gripper": "true"},
+            {
+                "arm_id": arm_id,
+                "load_gripper": "true",
+                "cpu_affinity": 0,
```

Comment:
should we add 1 more core? I don't know if the code below is correct.
```suggestion
                "cpu_affinity": [0, 1],
```

Reply:
It is correct. This most likely on most machines keep the code running on a single core, just jumping the execution between two v-cores. Tho I was not 100% how portable it will be. I bet 95% of the cases is should be portable

Reply:
mmmh the jump is not super great either.
Maybe we can start merging this and test it in prague?

Reply:
@kzorina said she will test it at 3 PM today. Afterwards we can merge it, I think. It's not conflicting with anything. So far I tested it on my laptop with fake hardware of ros2_control and it seemed to work

Reply:
@kzorina any news from the trials?

Reply:
@MaximilienNaveau @Kotochleb if I do in it regular user, I still run into the `communication_constraints_violation`, so doesn't fix the problem

Reply:
I think it is expected that it does not solve the problem.

Forcing a process to run on a single core does not make it run faster in general, unless you prevent other processes to run on this core. Thread affinity is different from CPU isolation.